scripts/main.py

```diff
@@ -105,7 +105,8 @@ def template_index():
                                 base_path, p) for p in plugin_folder_paths]
 
         # Get plugin folder names, in the specified order
-        if (region_json["pluginOrder"] is not None):
+        if ("pluginOrder" in region_json["pluginOrder"] and
```

Comment:
Typo, should be `if("pluginOrder" in region_json...)`

Reply:
Thanks!